components/runners/ambient-runner/ambient_runner/bridges/claude/corrections.py

```
@@ -282,7 +291,31 @@ async def log_correction_tool(args: dict) -> dict:
+            # Fire-and-forget POST to backend corrections endpoint
+            context = _get_session_context()
+            asyncio.ensure_future(
+                _post_correction_to_backend(
+                    session_name=context.get("session_name", _session_id),
+                    correction_type=correction_type,
+                    agent_action=agent_action,
+                    user_correction=user_correction,
+                    target_label=target_label,
+                    source=source,
+                )
+            )
```

Ruff (0.15.10)

[warning] Store a reference to the return value of `asyncio.ensure_future`

(RUF006)